share/functions/fish_vi_key_bindings.fish

```diff
@@ -17,14 +17,13 @@ function fish_vi_key_bindings --description 'vi-like key bindings for fish'
         set init_mode $argv[1]
     end
 
-    # Inherit default key bindings.
+    # Inherit shared key bindings.
```

Comment:
I like this wording since it makes it clearer that the bindings are shared by the two opposite bindings (i.e., emacs vs vi).